peachjam/templates/peachjam/_registries.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+{% if registries %}
+  <h4 class="my-4">Registries</h4>
```

Comment:
```suggestion
  <h4>Registries</h4>
```

otherwise the title is too far separated from the list.